builtin/credential/cert/path_certs.go

```diff
@@ -23,6 +23,10 @@ func pathListCerts(b *backend) *framework.Path {
 
 		HelpSynopsis:    pathCertHelpSyn,
 		HelpDescription: pathCertHelpDesc,
+		DisplayAttrs: &framework.DisplayAttributes{
+			Navigation: true,
```

Comment:
this boolean describes whether the item should have a tab in the navigation:

![image](https://user-images.githubusercontent.com/903288/88088330-12fa6780-cb3f-11ea-9371-6b24a3796c80.png)